### ISSUE TITLE
Imports duplicates on same data source

### ISSUE THREAD

**Issue by johnellison** [prosper-community/ImportScripts#3]
<img width="1440" alt="screen shot 2016-02-29 at 6 03 58 pm" src="https://cloud.githubusercontent.com/assets/5806135/13403833/d30b82e2-df0e-11e5-9c8c-9c07566657da.png">

@VirtualDmitriy I ran an import earlier today to grab the latest projects submitted from the form responses along with a couple new data sources. I noticed immediately that the import script was importing _all_ of the form response entries from the beginning so I stopped the script, removed the form responses from the data source and ran the import with the new sources.

Upon sorting through the data after the newest import, it looks like the form responses have been imported as duplicates throughout the entire database. My guess is when you ran the script last time it imported all of the form responses from the beginning (rather than skipping over the ones we've already imported).

I used a custom add-on to remove the duplicates (it wasn't that hard).

It's not a huge deal but we might need to fix this duplication bug before running another import of the form responses...

I can just run the de-duplicate add-on after each import until we figure out what this issue is. It's not a huge deal I just thought I'd open the issue so we can keep track of it.

**Comment by VirtualDmitriy** [comment]
@johnellison  Good catch this was accounted for in the script but I will take a look and see why it is no longer working the way it was before.
